support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -45,6 +50,21 @@ function mapDispatchToProps(dispatch: Dispatch<Action>) {
   };
 }
 
+
+const createSignInLink = (email: string, csrf: string) => {
+  const payload = { email };
+  fetch(routes.createSignInUrl, {
+    method: 'post',
+    headers: {
+      'Csrf-Token': csrf,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+  })
+    .then(response => response.json())
+    .then((data) => { window.location.href = data.signInLink; });
```

Comment:
Do we need to handle API errors in this function?

Reply:
Yes, we should 👍 thanks